lib/cob_index/macros/custom.rb

```diff
@@ -47,20 +47,18 @@ def extract_title_statement
           slash = "/"
 
           Traject::MarcExtractor.cached("245abcfgknps", alternate_script: false).collect_matching_lines(rec) do |field, spec, extractor|
```

Comment:
From reading the Jira ticket, I thought the plan was falling back to 245ATOZ where we don't have 245abcfgknps. I don't see any change/logic related to expanding subfields here.  Is the plan different now?

Reply:
@sensei100 this is just a refactor PR.  I was going to do an issue PR once I clean this up a bit and finish some discussion I'm having on the on the  BL-831 ticket... but this is not my attempt at fixing that.